tests/functional-sdk/test_sdk_sanity.py

```diff
@@ -171,6 +171,12 @@ def test_custom_gateway(vms, nets, init_dict):
             assert nets[net_name].gw() == net['gw']
 
 
+def test_vm_is_running(vms, vm_name):
+    vm = vms[vm_name]
```

Comment:
might be that the vm object is None. Check that

Reply:
How could it be? in which scenario?